galley/pkg/server/settings/args.go

```diff
@@ -186,8 +190,9 @@ func DefaultArgs() *Args {
 			Path:           defaultReadinessProbeFilePath,
 			UpdateInterval: defaultProbeCheckInterval,
 		},
-		Snapshots:       []string{metadata.Default, metadata.SyntheticServiceEntry},
-		TriggerSnapshot: metadata.Default,
+		Snapshots:        []string{metadata.Default, metadata.SyntheticServiceEntry},
+		TriggerSnapshot:  metadata.Default,
+		WatchedNamespace: "",
```

Comment:
You mention `meta_v1.NamespaceAll` in the comment for WatchedNamespace, should we use it here?

Reply:
wasn't sure if we needed to deprecate the ingress controller in pilot for a release first. let me know if there are users using this. 

if so, i can add a feature flag to the galley stuff and add a deprecation warning to pilot.
otherwise i'm happy to remove the controller from Galley.